tests/test_arrow.py

```diff
@@ -509,6 +527,17 @@ def test_isoformat(self):
 
         assert result == self.arrow._datetime.isoformat()
 
+    def test_isoformat_timespec(self):
+
+        result1 = self.arrow.isoformat(timespec="hours")
+        result2 = self.arrow.isoformat(timespec="microseconds")
+        result3 = self.arrow.isoformat(timespec="milliseconds")
+        result4 = self.arrow.isoformat(sep="x", timespec="seconds")
+        assert result1 == self.arrow._datetime.isoformat(timespec="hours")
+        assert result2 == self.arrow._datetime.isoformat()
+        assert result3 == self.arrow._datetime.isoformat(timespec="milliseconds")
+        assert result4 == self.arrow._datetime.isoformat(sep="x", timespec="seconds")
```

Comment:
Let's do `result = ...` then the assertion rather than numbering the results.

E.g.

```python
result = self.arrow.isoformat(timespec="hours")
assert result == self.arrow._datetime.isoformat(timespec="hours")
```

Reply:
Yes I agree. I tried option 2 since its more concise, but  the line got too long so linter ended up doing a different formatting for each of the asserts. So I went with option 1 and it does look cleaner and more readable.